packages/sources/svelte/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+	parser: '@babel/eslint-parser',
```

Comment:
What do you use for code formatting ? look line the indentation is double space compared to the React project